### PR TITLE
 [FLINK-20422] Remove site.baseurl from remaining .html templates 

### PR DESCRIPTION
**Pull request by dawidwys**
## What is the purpose of the change

The purpose is to remove the remaining occurences of `site.baseurl`.

## Brief change log

See the commit log for the commits.

## Verifying this change

Build the docs and click through them.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -88,6 +88,7 @@ previous_docs:
 exclude:
   - "build_docs.sh"
   - "check_links.sh"
+  - "spider.log"
```
Comment (sjwiesman) [inline docs/_config.yml:91]: +1

**Comment by sjwiesman** [comment]
I'm not entirely sure how to review this. It builds so +1 from me

**Comment by dawidwys** [comment]
@sjwiesman I understand it's impossible to review every single line of the PR. I'd say if you are fine with the concepts/general changes that each commit represents and it builds, we could merge it.

**Review by sjwiesman — APPROVED** [review]
Sounds good to me

**Pull request merged**